app/models/Backend.scala

```diff
@@ -1,5 +1,7 @@
 package models
 
+import akka.http.scaladsl.model.DateTime
+import cats.kernel.Comparison.GreaterThan
```

Comment:
These imports are unused and can be removed.

Reply:
The imports have been removed from the file